Diagnostics integration: fix typo (#35612)
home-assistant/home-assistant.github.io: source/_integrations/diagnostics.markdown

ha_integration_type: system
---

> The **Diagnostics** {% term integration %} provides a way to download diagnostic data from a {% term device %} or {% term integration %} to share it in issue reports. Sharing diagnostics data when reporting an issue allows developers to diagnose and fix your reported problem quicker.

Reviewer: ```suggestion
The **Diagnostics** {% term integration %} provides a way to download diagnostic data from a {% term device %} or {% term integration %} for sharing in issue reports. Sharing diagnostics data when reporting an issue allows developers to diagnose and fix your reported problem quicker.
```
